### PR TITLE
obsolete disorder involving pain

### LINKED ISSUE #3163: [Obsolete] disorder involving pain
**Mondo term (ID and Label)**
disorder involving pain
MONDO:0021668

**Reason for deprecation**
not useful term
(see[Mondo April2021 workshop]( https://docs.google.com/document/d/1KXqGAZgZUbK-VnzcomnMbfAEq7WEeGxDaQ-EK8UUQZI/edit))

From workshop:
**Discussions**: 
- not useful term:  all diseases will involve pain
- These terms have : “has major feature” = pain
- most of these terms come from a neurological pain syndrome

**AI: Obsolete, subclasses should be neurologic pain issues** 

Related to #2741

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fix #3163

### CODE REVIEW

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -435509,6 +435509,17 @@ def: "A disease characteristic in which the disease has a known cause." [https:/
 is_a: MONDO:0700004 {source="https://github.com/monarch-initiative/mondo/issues/3177", source="https://orcid.org/0000-0002-4142-7153"} ! idiopathic vs non-idiopathic
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700057
+name: neurological pain syndrome
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:435514]: I confirmed with Ada this should be renamed to neurological pain disorder

[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:435514]: Here is a suggested definition:
A nervous system disorder that has pain as a major feature.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
 xref: DOID:0060164 {source="MONDO:equivalentTo"}
 xref: MESH:D010146 {source="MONDO:otherHierarchy"}
 xref: SCTID:373673007 {source="MONDO:equivalentTo"}
 xref: UMLS:C1300028 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000001 {source="DOID:0060164/inferred", source="MONDO:Redundant", source="linkedlifedata"} ! disease or disorder
-intersection_of: MONDO:0000001 ! disease or disorder
-intersection_of: disease_has_major_feature HP:0012531 ! Pain
 relationship: excluded_subClassOf MONDO:0003117 {source="DOID:0060164", source="https://github.com/monarch-initiative/mondo/issues/130"} ! somatoform disorder
-relationship: realized_in GO:0019233 ! sensory perception of pain
+property_value: IAO:0000231 IAO:0000121
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:395570]: done

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
+is_a: MONDO:0700057 {source="MONDO:Redundant", source="https://orcid.org/0000-0002-4142-7153", source="linkedlifedata"} ! neurological pain syndrome
 intersection_of: MONDO:0005244 ! peripheral neuropathy
 intersection_of: disease_has_major_feature HP:0012531 ! Pain
 
 [Term]
 id: MONDO:0021668
-name: disorder involving pain
+name: obsolete disorder involving pain
 synonym: "pain disorder" EXACT [MONDO:0000691]
 xref: DOID:0060164 {source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:395565]: done

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
 xref: MESH:D010146 {source="MONDO:otherHierarchy"}
 xref: SCTID:373673007 {source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:395567]: done

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
 xref: UMLS:C1300028 {source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:395568]: done

File: src/ontology/mondo-edit.obo (hunk 1 above)
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:435514]: done

[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:435514]: done

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
 synonym: "pain disorder" EXACT [MONDO:0000691]
-xref: DOID:0060164 {source="MONDO:equivalentTo"}
+xref: DOID:0060164 {source="MONDO:obsoleteEquivalentTo"}
 xref: MESH:D010146 {source="MONDO:otherHierarchy"}
-xref: SCTID:373673007 {source="MONDO:equivalentTo"}
-xref: UMLS:C1300028 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000001 {source="DOID:0060164/inferred", source="MONDO:Redundant", source="linkedlifedata"} ! disease or disorder
-intersection_of: MONDO:0000001 ! disease or disorder
-intersection_of: disease_has_major_feature HP:0012531 ! Pain
+xref: SCTID:373673007 {source="MONDO:obsoleteEquivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:395567]: this should be MONDO:obsoleteEquivalent

not MONDO:obsoleteEquivalentTo

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -395554,23 +395554,23 @@ xref: ICD9:729.2 {source="MONDO:relatedTo", source="i2s"}
...
+xref: UMLS:C1300028 {source="MONDO:obsoleteEquivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:395568]: same as above

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -435509,6 +435509,18 @@ def: "A disease characteristic in which the disease has a known cause." [https:/
 is_a: MONDO:0700004 {source="https://github.com/monarch-initiative/mondo/issues/3177", source="https://orcid.org/0000-0002-4142-7153"} ! idiopathic vs non-idiopathic
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700057
+name: neurological pain disorder
+def: "A nervous system disorder that has pain as a major feature." []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:435515]: need a dbxref for the definition

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
a few minor changes still needed

File: src/ontology/mondo-edit.obo (hunk 8 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:435515]: done-added my ORCID (I hope that's an ok dbxref)

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
I am going to resolve this conflict now

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**